test/unit/lib/plugins/aws/remove/index.test.js

```diff
@@ -21,19 +21,28 @@ describe('AwsRemove', () => {
     let emptyS3BucketStub;
     let removeStackStub;
     let monitorStackStub;
+    let removeEcrRepositoryStub;
+    let checkIfEcrRepositoryExistsStub;
 
+    // TODO: Refactor to runServerless
```

Comment:
What is your opinion on refactoring this whole test suite to runServerless as a part of this Pr? I feel like it deserves another PR but on the other hand, it's pretty hard to write a meaningful test otherwise

Reply:
Totally, let's do it! (and in other PR as you propose)